### ISSUE TITLE
Show byte arrays as a hex string in Debug impls, not an expanded array

### ISSUE THREAD

**Issue by teor2345** [RustCrypto/signatures#723]
When I debug-format the Signature type, I get some very long output (see below). But it would be much easier to read and copy shorter hex-encoded output.

Expected output:
```
sig: ed25519::Signature {
    R: "e8...",
    s: "77...",
}
```

Actual output:
```
sig: ed25519::Signature {
    R: [
        232,
        35,
        39,
        173,
        39,
        51,
        101,
        13,
        82,
        4,
        201,
        65,
        153,
        155,
        75,
        114,
        8,
        44,
        128,
        70,
        20,
        142,
        169,
        129,
        27,
        35,
        129,
        240,
        229,
        20,
        112,
        240,
    ],
    s: [
        119,
        225,
        75,
        170,
        146,
        238,
        92,
        78,
        105,
        184,
        143,
        18,
        153,
        212,
        246,
        171,
        34,
        106,
        132,
        183,
        50,
        226,
        59,
        202,
        111,
        193,
        180,
        80,
        34,
        219,
        107,
        243,
    ],
},
```

**Comment by tarcieri** [comment]
That'd probably be good, although I'd note you can already get hex via the `Display` impl

**Comment by tarcieri** [comment]
Fixed in #747 which was released in `ed25519` v2.2.3
